chain/client/src/test_utils/test_env.rs

```diff
@@ -418,23 +424,37 @@ impl TestEnv {
             client.epoch_manager.account_id_to_shard_id(&account_id, &head.epoch_id).unwrap();
         let shard_uid = client.epoch_manager.shard_id_to_uid(shard_id, &head.epoch_id).unwrap();
         let last_chunk_header = &last_block.chunks()[shard_id as usize];
-        let response = client
-            .runtime_adapter
-            .query(
-                shard_uid,
-                &last_chunk_header.prev_state_root(),
-                last_block.header().height(),
-                last_block.header().raw_timestamp(),
-                last_block.header().prev_hash(),
-                last_block.header().hash(),
-                last_block.header().epoch_id(),
-                &QueryRequest::ViewAccount { account_id },
-            )
-            .unwrap();
-        match response.kind {
-            QueryResponseKind::ViewAccount(account_view) => account_view,
-            _ => panic!("Wrong return value"),
+
+        for i in 0..self.clients.len() {
+            let tracks_shard = client
```

Comment:
This should be `self.clients[i]` not `client`

Reply:
It doesn't actually matter, because I'm querying using epoch_manager which would give the same result on every client. If I had used the ShardTracker then it would take local tracking config into account.